spec/generators/rspec/generators/generator_generator_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'generators/rspec/generators/generator_generator'
+require 'support/generators'
+
+ RSpec.describe Rspec::Generators::GeneratorsGenerator, :type => :generator do
+   setup_default_destination
+
+   describe "generator specs" do
+     subject(:generator_spec) { file("spec/generator/posts_generator_spec.rb") }
+     describe "are generated independently from the command line" do
+       before do
+         run_generator %w(posts)
+       end
+       describe "the spec" do
```

Comment:
You can collapse this into the docs strings below:
it "creates the spec file"
it "includes the rails helper in the spec file"
it "includes the generator type in the metadata"